vcpkg/get_started/overview.md

```diff
@@ -4,16 +4,16 @@ description: Learn about how the vcpkg C/C++ package manager can help you acquir
 ms.topic: overview
 author: AugP
 ms.author: aupopa
-ms.date: 01/10/2024
+ms.date: 07/11/2024
 ---
 
 # vcpkg overview
 
-vcpkg is a free and open-source C/C++ package manager maintained by Microsoft and the C++ community. Launched in 2016, it helps developers migrate their projects to newer versions of Visual Studio. vcpkg has evolved into a cross-platform tool used by developers on Windows, macOS, and Linux. vcpkg has a large registry of open-source libraries and enterprise-ready features designed to facilitate your development process with support for any build and project systems. vcpkg is a C++ tool at heart and is written in C++ with scripts in CMake. It is designed from the ground up to address the unique pain points of the C/C++ developer experience.
+vcpkg is a free and open-source C/C++ package manager maintained by Microsoft and the C++ community. vcpkg runs on Windows, macOS, and Linux (see [Supported hosts](../concepts/supported-hosts.md)) and includes a large, curated registry of open-source libraries and enterprise-ready features designed to facilitate your development process with support for [any build and project system](../concepts/build-system-integration.md). With vcpkg, you can build libraries from source to target virtually any platform or architecture using the customizable [triplets experience](../concepts/triplets.md). vcpkg is a C++ tool at heart and is written in C++ and CMake scripts. It is designed from the ground up to address the unique pain points of the C/C++ developer experience.
```

Comment:
Sentences in this paragraph seem to run longer than they should.  We should consider reformatting it into something nicer.

```suggestion
vcpkg is a free and open-source C/C++ package manager maintained by Microsoft and the C++ community that runs on Windows, macOS, and Linux (see [Supported hosts](../concepts/supported-hosts.md)). vcpkg is a C++ tool at heart and is written in C++ and CMake scripts. It is designed from the ground up to address the unique pain points of the C/C++ developer experience.

Adopt vcpkg to enjoy these benefits:

* A large, curated, and ever-growing registry of open-source libraries.
* Enterprise-ready features designed to facilitate your development workflows.
* Support for any [build and project system](../concepts/build-system-integration.md) with CMake and MSBuild projects supported as first-class experiences.
* Target virtually any platform or architecture using [customizable triplets](../concepts/triplets.md)
```